packages/components/news/6069.breaking

```diff
@@ -0,0 +1,6 @@
+Removed the `FlattenToAppURLProvider` since is no longer needed @sneridagh
+The components in here are meant to to need it.
+Refactored the `Link` component to not use it since `react-aria-components` use the React Client Routing facilities that can be injected in the React tree.
```

Comment:
```suggestion
Refactored the `Link` component to not use it, since `react-aria-components` uses the React Client Routing facilities that can be injected into the React tree.
```